deploy-docker-compose.yml

```
@@ -0,0 +1,322 @@
+
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    networks:                 # ✅ 이거 추가
+    - app-network
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)